src/Propel/Generator/Platform/DefaultPlatform.php

```diff
@@ -1442,6 +1442,26 @@ public function getDateFormatter()
         return 'Y-m-d';
     }
 
+    /**
+     * Get the default On Delete behavior for foreign keys when not explicity set.
+     *
+     * @return string
+     */
+    public function getDefaultForeignKeyOnDeleteBehavior()
```

Comment:
here also return type could be added etc for new code

Reply:
Done